src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -119,8 +119,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Include="-lstdc++" Condition="'$(LinkStandardCPlusPlusLibrary)' == 'true'" />
       <LinkerArg Include="-ldl" />
       <LinkerArg Include="-lobjc" Condition="'$(_IsApplePlatform)' == 'true'" />
-      <LinkerArg Include="-lswiftCore" Condition="'$(_IsApplePlatform)' == 'true'" />
-      <LinkerArg Include="-lswiftFoundation" Condition="'$(_IsApplePlatform)' == 'true'" />
+      <LinkerArg Include="-lswiftCore" Condition="'$(_targetOS)' == 'osx'" />
+      <LinkerArg Include="-lswiftFoundation" Condition="'$(_targetOS)' == 'osx'" />
```

Comment:
Do we still need line: https://github.com/dotnet/runtime/pull/87216/files#diff-a8c821001409a0c4d6589a59e663d4d07e53b0127b345f17640410828c580b37R127

Reply:
No. Will fix it when I get back to computer.